api/interceptor/observer_test.go

```diff
@@ -0,0 +1,87 @@
+package interceptor_test
```

Comment:
Might be good to add streaming test that uses `WrapStreamingHandler`.

Reply:
Added